eZ/Bundle/EzPublishCoreBundle/EventListener/ConsoleCommandListener.php

```diff
@@ -42,9 +48,10 @@ class ConsoleCommandListener implements EventSubscriberInterface, SiteAccessAwar
     /**
      * ConsoleCommandListener constructor.
      */
-    public function __construct($defaultSiteAccessName, EventDispatcherInterface $eventDispatcher)
+    public function __construct($defaultSiteAccess, $siteAccessList, EventDispatcherInterface $eventDispatcher)
```

Comment:
Why did you change `$defaultSiteAccessName` to `$defaultSiteAccess`? Usually SiteAccess is an object, so this makes it confusing here (same for property name).
Besides, `$siteAccessList` should be  type hinted as array :wink: